ci/test/00_setup_env_i686_centos.sh

```diff
@@ -12,6 +12,7 @@ export CI_IMAGE_NAME_TAG=quay.io/centos/centos:stream8
 export CI_BASE_PACKAGES="gcc-c++ glibc-devel.x86_64 libstdc++-devel.x86_64 glibc-devel.i686 libstdc++-devel.i686 ccache libtool make git python38 python38-pip which patch lbzip2 xz procps-ng dash rsync coreutils bison"
 export PIP_PACKAGES="pyzmq"
 export GOAL="install"
+export NO_WERROR=1
```

Comment:
While it is obvious how old a compiler is in other tasks with `NO_WERROR=1`, that is not the case in here. Maybe comment a compiler's version in this task?

Reply:
Added GCC 8 here.